src/GafferSceneUI/StandardLightVisualiser.cpp

```diff
@@ -571,8 +572,14 @@ IECoreGL::ConstRenderablePtr StandardLightVisualiser::environmentSphere( const I
 	IECore::CompoundObjectPtr parameters = new CompoundObject;
 	parameters->members()["lightMultiplier"] = new Color3fData( color );
 	parameters->members()["previewOpacity"] = new FloatData( 1 );
-	parameters->members()["mapSampler"] = new StringData( textureFileName );
 	parameters->members()["defaultColor"] = new Color3fData( Color3f( textureFileName == "" ? 1.0f : 0.0f ) );
+	parameters->members()["mapSampler"] = new StringData( textureFileName );
+
+	if( textureMaxResolution )
+	{
+		parameters->members()["mapSampler_maxResolution"] = textureMaxResolution->copy();
```

Comment:
What's the failure mode for this when used with a Cortex without support? Is it just a warning, or does it break the drawing in some way? Just wondering if we can merge in anticipation of the Cortex update or if it should wait.

Reply:
I couldn't see any problems. Didn't even see a warning.